src/components/Form/Fields/CreatableSelect.js

```diff
@@ -124,10 +136,19 @@ class Select extends PureComponent {
           // a round about way, and still allow us to use the `touched` property.
           onBlur={this.handleBlur}
           blurInputOnSelect={false}
+          isValidNewOption={(option) => {
+            // Logic for showing the "create" option is simple:
+            // If the input value matches an item in the options collection,
+            // don't show. Otherwise, show.
+            const alreadyExists = options.some(({ variableLabel }) => variableLabel === option);
```

Comment:
`variableLabel` should this just be `label`, or at least `{ label: variableLabel }`? The UI currently allows you to create variable with the same name (even with matching case).

Reply:
Yes, sorry. Major doh from me on this one. Just trying to hurry it through! Will fix now.